store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -50,26 +50,50 @@
   private String tableName;
   private boolean isTransactionalTable = true;
 
+  /**
+   * Construct a CarbonReaderBuilder with table path and table name
+   *
+   * @param tablePath table path
+   * @param tableName table name
+   */
   CarbonReaderBuilder(String tablePath, String tableName) {
     this.tablePath = tablePath;
     this.tableName = tableName;
   }
 
+  /**
+   * Configure the projection column names of carbon reader
+   *
+   * @param projectionColumnNames projection column names
+   * @return CarbonReaderBuilder
+   */
   public CarbonReaderBuilder projection(String[] projectionColumnNames) {
     Objects.requireNonNull(projectionColumnNames);
     this.projectionColumns = projectionColumnNames;
     return this;
   }
 
+  /**
+   * Configure the transactional status of table
+   *
+   * @param isTransactionalTable whether is transactional table or not
```

Comment:
update the default value and mention that if false, read from flat folder. 

Else user will not know what is this API for.

Reply:
ok, Done